### ISSUE TITLE
Barcode Positioning 

### ISSUE THREAD

**Issue by applsais** [sdaps/sdaps#264]
Is there a way to separately move the questionnaire id and survey id? We have figured out how to move them vertically and horizontally but they move together and we want to change that. We want them to be in separate positions. Additionally, would it be possible to rotate the barcodes? Finally, would this affect the scanning? We would like an answer as soon as possible. Thank you!

**Comment by benzea** [comment]
I hope you are using the latest version of the class (i.e. 1.9.x)?

There is a scope around the barcode, and the positioning happens relative to the corner bottom corner marks. So yeah, one can just shift it around and even change the anchor point.

And yes, you likely need to modify the recognition process. i.e. `recognize/qrcode.py`, `recognize/code128.py`, etc. hard-code the location.

**Comment by applsais** [comment]
How would I go about shifting the anchor point and rotating? Also, I have been having issues with SDAPs not allowing me to put images on my documents. How would I fix that as well? Thank you!

**Comment by applsais** [comment]
We have a very simple form with a couple of multiple-choice questions. However, we want to move the barcodes which are placed at the bottom to be on the middle right on top of each other where there is whitespace. How would we do that?

**Comment by applsais** [comment]
Would it be possible to quickly show our issue to you over a video conference just for a couple of minutes if you have some time?  We'd really appreciate that.

**Comment by benzea** [comment]
Yeah, could do a quick call some time probably. Not sure when I will be free on the weekend, but I am sure we can figure something out next week.

**Comment by benzea** [comment]
Sure, I am reasonably flexible overall. Maybe Wednesday around 16:00 UTC? (I just avoided Monday/Tuesday because it is a bit more convenient for me, Tuesday would be fine in the afternoon and Monday I could probably manage in the evening only though).

**Comment by applsais** [comment]
Wednesday should work! Thank you so much.

**Comment by benzea** [comment]
Wait, did we talk about yesterday or next week? I don't mind, but I got confused (and, to be honest, forgot about it yesterday as I was quite busy with an emergency). Also, do you want to setup a meeting and share the link?

Maybe send me a mail directly, probably easier than doing it here (benjamin@sipsolutions.net).
